initfiles/bash/etc/init.d/hpcc_common.in

```diff
@@ -468,6 +468,20 @@ startCmd() {
         fi
     fi
 
+    # apply user limits we explicitly added ...
+    myid=$(id -u 2>/dev/null)
+    if [[ "${myid}" = "0" ]] ; then
+        nofile_user=$(su - ${user} -c 'bash -c "ulimit -Hn"' 2>/dev/null)
+        rc=$?
+        if [[ $rc -ne 0 ]] ; then
+            nofile_user=$(grep 'hpcc* hard* nofile' /etc/security/limits.conf 2>/dev/null | awk '{print $4}')
+        fi
+        if [[ -n "${nofile_user}" ]] ; then
+            ulimit -Hn ${nofile_user} >/dev/null 2>&1
+        fi
+    fi
+
+    # check limits are at least as high as the minimums required ...
     limits=(
```

Comment:
NOTE: or should we just change these values to what is found in /etc/security/limits.conf or limits.d/* ?